app/api/helpers/order.py

```diff
@@ -73,9 +73,10 @@ def create_pdf_tickets_for_holder(order):
             save_to_db(holder)
 
         # create order invoices pdf
-        order_ticket_info = OrderTicket.query.filter_by(order_id=order.id).one()
+        order_tickets = OrderTicket.query.filter_by(order_id=order.id).all()
```

Comment:
There's no deleted_at in OrderTicket model right? If there is `deleted_at=None`

Reply:
@iamareebjamal Done.